src/ches/emailHandlersPublic.ts

```diff
@@ -1,37 +1,42 @@
 import {
   generateEmailTemplatePublicData,
-} from "./emailHelpers.js"
-import chesService from "./index.js";
-import swig from "swig";
+  contactChanged,
+  budgetChanged,
+} from './emailHelpers.js';
+import chesService from './index.js';
+import swig from 'swig';
 import {
   RequestType,
   RequestDecision,
-} from "../__generated__/resolvers-types.js";
-import {
-  adminPublicEmails,
-} from "./emailConstants.js";
+} from '../__generated__/resolvers-types.js';
+import { adminPublicEmails } from './emailConstants.js';
+
+const requestedChanges = (project, requestedProject) =>
```

Comment:
Can we rename the function to be more descriptive?
As the function returns a Boolean type, we can rename it to indicate the type. e.g. `checkIfProjectChanged`.